front-end/src/app/contacts/contact-list/contact-list.component.html

```diff
@@ -77,33 +77,32 @@ <h5 class="m-0">Manage Contacts</h5>
         <th pSortableColumn="occupation" id="occupation-column" role="columnheader">
           Occupation <p-sortIcon field="occupation"></p-sortIcon>
         </th>
-        <th id="buttons" role="columnheader"></th>
+        <th id="buttons" role="columnheader" style="text-align: center">Delete</th>
```

Comment:
It _might_ be worth using the `.text-center` class from shared/scss/classes/_text.scss instead.  It's a very small thing, though

Reply:
FYI: That scss directory in shared is legacy styling that is going to be eliminated as soon as the current login page goes away. Also, unfortunately, the style is being used because that was the only way to enforce the centering due to aggressive and contrary parent element styling.